DependencyInjection/Configuration.php

```diff
@@ -69,12 +75,33 @@ public function getConfigTreeBuilder()
                             ->then(function ($v) {
                                 return ['id' => $v];
                             })
+                            ->ifTrue(function ($keys) {
+                                foreach ($keys as $key => $value) {
+                                    if (isset($keys['$'.$key])) {
+                                        return true;
+                                    }
+                                }
+                                return false;
+                            })
+                            ->thenInvalid('Duplicate variable in %s. Prefer use var with $ if var is reserved properties.')
                         ->end()
                         ->children()
                             ->scalarNode('id')->isRequired()->end()
+                            ->scalarNode('first_name')
+                                ->setDeprecated($this->symfonyVersion === 4 ? 'The "%node%" option at path "%path%" is deprecated. Use "$first_name" instead.':'gordalina/mixpanel-bundle', '3.2', 'The "%node%" option at path "%path%" is deprecated. Use "$first_name" instead.')
```

Comment:
Instead of duplicating the deprecation message and package name each time, maybe you can use a private method that will do the job, see https://github.com/doctrine/DoctrineMigrationsBundle/pull/357.